### PR TITLE
Added booking payments summary notifications

### PR DESCRIPTION
**Pull request by mingazhev**

### CODE REVIEW

File: Api/Controllers/AgentControllers/InternalBookingsController.cs [hunk 1]
```diff
@@ -182,12 +182,31 @@ public async Task<IActionResult> NotifyBookingsSummaryAdministrator()
...
+        [ServiceAccountRequired]
+        public async Task<IActionResult> NotifyBookingsPaymentsSummaryAdministrator()
+        {
+            // TODO: Ad-hoc solution, change to more appropriate
+            var result = await _bookingMailingService.SendBookingsPaymentsSummaryAdministrator();
+            var message = result.IsSuccess
+                ? "Administrator payments report was sent successfully"
+                : $"Error: {result.Error}";
+            
+            return Ok(new BatchOperationResult(message, result.IsFailure));
```
Comment (kirillta) [inline Api/Controllers/AgentControllers/InternalBookingsController.cs:207]: Why do you return errors with OK code?

File: Api/Services/Mailing/IBookingMailingService.cs [hunk 2]
```diff
@@ -20,5 +20,7 @@ public interface IBookingMailingService
         Task<Result<string>> SendBookingReports(int agencyId);
 
         Task<Result> SendBookingsAdministratorSummary();
+        
+        Task<Result> SendBookingsPaymentsSummaryAdministrator();
```
Comment (kirillta) [inline Api/Services/Mailing/IBookingMailingService.cs:24]: SendBookingsPaymentsSummary**To**Administrator?

**Review by kirillta — CHANGES_REQUESTED** [review]

File: Api/Services/Mailing/IBookingMailingService.cs (hunk 2 above)
Comment (mingazhev) [reply Api/Services/Mailing/IBookingMailingService.cs]: Fixed

File: Api/Controllers/AgentControllers/InternalBookingsController.cs (hunk 1 above)
Comment (mingazhev) [reply Api/Controllers/AgentControllers/InternalBookingsController.cs]: Fixed

**Review by kirillta — APPROVED** [review]

**Pull request merged**